migrations/5_deploy_child_contracts.js

```diff
@@ -7,9 +7,11 @@ const ChildChain = artifacts.require('ChildChain')
 const MRC20 = artifacts.require('MRC20')
 
 module.exports = async function(deployer, network, accounts) {
-  if (deployer.network !== 'bor') {
-    return
-  }
+  // if (deployer.network !== 'bor') {
```

Comment:
This comment is intentional?

Reply:
Yeah. We wanted to deploy contracts on bordevnet. In this case deployer.network must be "bordevnet".
The commented code will return the function and migration code will not run. So I commented it.

If this commented code is needed part, then we can add one more condition.
if (deployer.network !== 'bor' && deployer.network !== 'bordevnet') {
   return
}